### PR TITLE
Update bioext tools to 0.19.7 and sanitize input

### PR DESCRIPTION
**Pull request by davebx**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR updates an existing tool or tool collection

This update, along with upgrading to bioext 0.19.7, adds a simple input filter to the sequences being passed in, removing characters that might affect how `bealign` aligns sequences to a reference.

### CODE REVIEW

File: tools/bioext/bealign.xml [hunk 1]
```diff
@@ -83,22 +87,22 @@
...
+            <output name="output" file="bealign-out2.bam" compare="sim_size" delta="8" />
         </test>
         <test>
             <param name="input" ftype="fasta" value="bealign-in2.fa" />
             <param name="reference_type" value="dataset" />
             <param name="expected_identity" value="0.9" />
             <param name="score_matrix" value="BLOSUM62" />
             <param name="reference" ftype="fasta" value="bealign-in-ref-2.fa" />
-            <output name="output" file="bealign-out3.bam" />
+            <output name="output" file="bealign-out3.bam" compare="sim_size" delta="8" />
```
Comment (bgruening) [inline tools/bioext/bealign.xml:105]: BAM files can be compared with diff, they will be unpacked. So here is probably something wrong, no?

File: tools/bioext/bealign.xml [hunk 2]
```diff
@@ -4,10 +4,14 @@
...
         <import>macros.xml</import>
     </macros>
-    <expand macro="requirements" />
+    <expand macro="requirements">
+        <requirement type="package" version="1.70">biopython</requirement>
+    </expand>
     <version_command>bealign --version</version_command>
     <command detect_errors="exit_code">
     <![CDATA[
+    cat '$input' | awk '{ if (\$0 ~ "^[^>]") {a = gensub(/[^ACGTURYKMSWBDHVNacgturykmswbdhvn?-]/, "", "g"); } else {a=gensub(/[^>A-Za-z0-9_]/, "_", "g"); }; print a } ' |
```
Comment (bgruening) [inline tools/bioext/bealign.xml:13]: can you please document this as a comment here and also in the help of the tool. I think its unexpected that this tool is changing the file, no?

File: tools/bioext/bealign.xml [hunk 3]
```diff
@@ -4,10 +4,14 @@
     <macros>
         <import>macros.xml</import>
     </macros>
-    <expand macro="requirements" />
+    <expand macro="requirements">
+        <requirement type="package" version="1.70">biopython</requirement>
```
Comment (bgruening) [inline tools/bioext/bealign.xml:8]: where do you use biopython? Is that an issue with bioext conda package? Should we fix the conda package instead?

File: tools/bioext/bealign.xml (hunk 2 above)
Comment (bgruening) [reply tools/bioext/bealign.xml:11]: @davebx :)

Comment (davebx) [reply tools/bioext/bealign.xml:18]: How's this?

**Review by bgruening — APPROVED** [review]

**Pull request merged**